server/src/main/java/server/BazelServices.java

```diff
@@ -188,16 +190,17 @@ public void connect(LanguageClient client) {
 
     @Override
     public CompletableFuture<Either<List<CompletionItem>, CompletionList>> completion(CompletionParams completionParams) {
-        String item = "THIS IS MY FAKE ITEM";
-
-        CompletionItem completionItem = new CompletionItem(item);
-        completionItem.setKind(CompletionItemKind.Folder);
-        completionItem.setInsertText(item);
-        completionItem.setTextEdit(new TextEdit(new Range(completionParams.getPosition(), new Position(completionParams.getPosition().getLine(), completionParams.getPosition().getCharacter() + item.length())), item));
-        logger.info("Added item: " + completionItem);
-        completionItems.add(completionItem);
-
-        return CompletableFuture.completedFuture(Either.forRight(new CompletionList(completionItems)));
+//        String item = "THIS IS MY FAKE ITEM 2";
+//
+//        CompletionItem completionItem = new CompletionItem(item);
+//        completionItem.setKind(CompletionItemKind.Folder);
+//        completionItem.setInsertText(item);
+//        completionItem.setTextEdit(new TextEdit(new Range(completionParams.getPosition(), new Position(completionParams.getPosition().getLine(), completionParams.getPosition().getCharacter() + item.length())), item));
+//        logger.info("Added item: " + completionItem);
+//        completionItems.add(completionItem);
+//
+//        return CompletableFuture.completedFuture(Either.forRight(new CompletionList(completionItems)));
```

Comment:
Is there any reason to keep this commented out proof of concept code? 
If not we should probably just remove it from the code base.

Reply:
I'll remove it in the next PR